### PR TITLE
Show node names on reservation calendar

### PR DESCRIPTION
**Pull request by jakecoll**
This replaces node uuids on reservation calendar with node names, which are more human readable. Implementation also requires adding node names to blazar host extra capabilities.

### CODE REVIEW

File: blazar_dashboard/api/client.py [hunk 1]
```diff
@@ -363,6 +364,31 @@ def extra_capability_names(request):
...
+    cursor = get_cursor_for_request(request)
+    sql = '''\
+    SELECT
+        id, computehost_id, capability_name, capability_value
+    FROM
+        computehost_extra_capabilities
+    WHERE
+        capability_name = %s
+    '''
+    cursor.execute(sql, [name])
```
Comment (diurnalist) [inline blazar_dashboard/api/client.py]: This is getting us away from our goal of reducing DB access. Can't we get this from the Blazar host list?

**Review by diurnalist — CHANGES_REQUESTED** [review]
Unfortunately, adding new DB calls is a non-starter, we are getting closer to reducing this dependency and I don't want to add more things we will have to undo later :(

File: blazar_dashboard/api/client.py (hunk 1 above)
Comment (diurnalist) [reply blazar_dashboard/api/client.py]: Err, I see that this isn't even called. What is it for?

Comment (jakecoll) [reply blazar_dashboard/api/client.py]: Meant to remove that. Played with that first, but can do with host_list. Updates pushed.

**Review by diurnalist — APPROVED** [review]
Great! Looks good

**Pull request merged**